### DISCUSSION TITLE
Allow to view corrected expression in ggplot after projection

### DISCUSSION THREAD

**Discussion by drorbar** [tanaylab/MCView#170] [Ideas]
Currently after we do projecting we get a new layer with the genes fractions after the projection correction,
While you can see the correction factor on the query page, when you look at the ggplot, you see the observed values.
Having a toggle of "corrected\ original" that will change the source of such values might be a good idea, in many cases you want to compare expression to the atlas and see them on the same scale.

**Comment by aviezerl** [comment]
done
